app/main/views/buyers.py

```diff
@@ -496,22 +516,13 @@ def award_or_cancel_brief(framework_slug, lot_slug, brief_id):
                 # We should never get here as the form validates the answers against the available choices.
                 abort(500, "Unexpected answer to award or cancel brief")
 
-    breadcrumbs = get_briefs_breadcrumbs([{
-        "label": brief['title'],
-        "link": url_for(
-            ".view_brief_overview",
-            framework_slug=brief['frameworkSlug'],
-            lot_slug=brief['lotSlug'],
-            brief_id=brief['id']
-        )
-    }])
-
     return render_template(
         "buyers/award_or_cancel_brief.html",
         brief=brief,
         form=form or AwardOrCancelBriefForm(brief),
         errors=errors,
-        breadcrumbs=breadcrumbs
+        breadcrumbs=breadcrumbs,
+        already_awarded=False,
```

Comment:
Could reduce the duplication of the `render_template` from above with:
```
if brief['status'] in ["awarded", "cancelled", "unsuccessful"]:
    return render_template(
        "buyers/award_or_cancel_brief.html",
        brief=brief,
        form=None,
        errors=None,
        breadcrumbs=breadcrumbs,
        already_awarded=True if if brief['status'] in ["awarded", "cancelled", "unsuccessful"] else False
    ), 200
```

Reply:
Well... not quite - that will fail the earlier 404 check on `if brief['status'] != 'closed'`. Which I suppose we could change to check for live/withdrawn/draft instead but if we end up adding new statuses that'll need updating. 

How about I keep the separate render_template statement, but take out the unnecessary `form=None` and `errors=None`? Shorter and sweeter.

Reply:
:man_shrugging:  I constructed that whole view to build up the params needed for the response based on the statuses or fail early. That's why `form` and `errors` are already respectively `None` and `{}` at the top of the view.

```
if brief['status'] in ["awarded", "cancelled", "unsuccessful"]:
    already_awarded = True
elif brief['status'] != 'closed':
    abort(404)
else:
    already_awarded = False
    if request.method == "POST":
        form = AwardOrCancelBriefForm(brief, request.form)
        ...
    else:
        form = AwardOrCancelBriefForm(brief)
breadcrumbs = ....
return render_template(
    "buyers/award_or_cancel_brief.html",
    brief=brief,
    form=form,
    errors=errors,
    breadcrumbs=breadcrumbs,
    already_awarded=already_awarded
), 200 if not errors else 400
```
Should work with really minimal changes

Reply:
That feels better to me - avoiding the POST for the non-closed status. OK, will give that a whirl!